db/blip_handler.go

```diff
@@ -249,6 +249,15 @@ type sendChangesOptions struct {
 	ignoreNoConflicts bool
 }
 
+type changesDeletedFlag uint
+
+const (
```

Comment:
I think it's more idiomatic go to use `1 << iota` for this scenario, isn't it?  Like we're doing for things like LogEntry.Flags?  So something like
```
const (
	// Bitfield flags used to build changes deleted property below
	changesDeletedFlagDeleted changesDeletedFlag = 1 << iota
	changesDeletedFlagRevoked 
	changesDeletedFlagRemoved 
)
```

Reply:
I suggested in an above comment explicit values, given the source of these values is the replication protocol, rather than the Go code, to avoid any future additions from changing the values.

Reply:
I see what you mean - I think that's reasonable.  In that case, what do you think about 0x01, 0x02, 0x04 instead of the 0b representations?   I think gocb follows that approach for flags, and I haven't actually seen the 0b format used previously.

Reply:
I personally prefer the binary representation as it makes it obvious it's a bit flag that has each different bit set. 
I can switch it though if you think it'd be better.